code/game/gamemodes/blob/theblob.dm

```diff
@@ -182,11 +184,7 @@
 		A.blob_act(src) //also hit everything in the turf
 
 	if(make_blob) //well, can we?
-		var/obj/structure/blob/B = new /obj/structure/blob/normal(src.loc)
-		if(controller)
-			B.overmind = controller
-		else
-			B.overmind = overmind
+		var/obj/structure/blob/B = new /obj/structure/blob/normal(src.loc, overmind)
```

Comment:
should still be controller

Reply:
well, `controller ? controller : overmind` but you get it

Reply:
`(controller || overmind)` nobody remembers short circuiting for non Boolean applications